src/DatasetAPI/Datasets.jl

```diff
@@ -282,7 +288,8 @@ function toaxis(dimname, g, offs, len)
     aratts = get_var_attrs(g, dimname)
     if match(r"^(days)|(hours)|(seconds)|(months) since",lowercase(get(aratts,"units",""))) !== nothing
         tsteps = try
-            timedecode(ar[:], aratts["units"], lowercase(get(aratts, "calendar", "standard")))
+            dec = timedecode(ar[:], aratts["units"], lowercase(get(aratts, "calendar", "standard")), prefer_datetime=false)
+            round_datetime.(dec)
```

Comment:
This is breaking because it changes the  output type to a DateTime. This function should only be called when the prefer_datetime keyword is true. 
This is also the cause of the doc build failure. 
```suggestion
            prefer_datetime && round_datetime.(dec)
```

Reply:
unfortunately,
 
```
failed to run `@example` block in docs/src/UserGuide/select.md:45-49
│ ```@example subset
│ using CFTime
│ time1 = DateTime360Day(2001,01,16)
│ tos[time = At(time1)]
│ ```
│   exception =
│    promotion of types CFTime.DateTime360Day{CFTime.Period{Int64, Val{1}(), Val{-3}()}, Val{(1900, 1, 1)}()} and DateTime failed to change any arguments
│    Stacktrace:
│      [1] error(::String, ::String, ::String)
```